ConsoleFollower/Program.cs

```diff
@@ -26,8 +40,20 @@ public class Program {
     // or go to https://www.sec.gov/edgar/searchedgar/companysearch.html and put the name of the investor you are interested in
     public static void Main(string[] args)
     {
-        if (args.Count() != 1) throw new Exception("Usage: Follower Cik");
-        var result =  GuruLoader.FetchDisplayPortfolio(args[0]).Result;
-        Console.WriteLine(DisplayPortToString(result));
+        var banner = "Usage: Follower Cik [-All]";
+        if (args.Count() > 2) throw new Exception(banner);
+        if (args.Count() == 2 && args[1] != "-All") throw new Exception(banner);
+
+        if(args.Count() == 1) {
+            var result = GuruLoader.FetchDisplayPortfolio(args[0]).Result;
+            Console.WriteLine(DisplayPortToString(result));
+        } else {
+            // Printing Portfolio summary at both start and bottom
+            var result = GuruLoader.FetchFullPortfolioData(args[0]).Result;
+            Console.WriteLine(DisplayPortToString(result.Portfolio));
+            Console.WriteLine(DisplayHistory(result.CompaniesHistory));
+            Console.WriteLine(DisplayPortToString(result.Portfolio));
```

Comment:
the listing gets long and it's useful to be able to refer to the full portfolio on both sides of it. But it looks odd for a demo app, so changed it.